.github/workflows/snyk.yaml

```diff
@@ -32,8 +33,19 @@ jobs:
           helm repo add bitnami https://charts.bitnami.com/bitnami
           helm repo add backstage https://backstage.github.io/charts
           helm repo update
-          helm dependency build ./charts/${{ matrix.chart }}
-          helm template ./charts/${{ matrix.chart }} --output-dir ./output/${{ matrix.chart }}
+
+          if [ "${{ matrix.chart }}" = "backstage-orchestrator" ]; then
```

Comment:
Instead of an `if` (which could be hard to maintain if we want to scan other combinations in the future), would you update the matrix configuration instead, such that each configuration specifies the parameters to pass to `helm template`? See https://github.com/redhat-developer/rhdh-local/blob/main/.github/workflows/test.yml#L35-L41 as an inspiration.